src/sys_stubs.c

```diff
@@ -0,0 +1,64 @@
+#include <stdint.h>
+#include <sys/stat.h>
+#include <stdbool.h>
+//doing the minimal amount possible to not break
+
+void* delugeAlloc(unsigned int requiredSize, bool mayUseOnChipRam);
+
+void* _sbrk(int incr) {
+	static unsigned char* heap = NULL;
+	static unsigned char* end_heap = NULL;
+	unsigned char* prev_heap;
+
+	if (heap == NULL) {
+		heap = delugeAlloc(10000, false);
```

Comment:
Can we do this with a linker section instead?

Reply:
Then it'll always be present, currently this is never called and only exists to do something better than corrupt memory in case of accidents